core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -124,6 +127,7 @@
             List<ClientAddressSource> clientAddressSources,
             Predicate<InetAddress> clientAddressTrustedProxyFilter,
             Predicate<InetAddress> clientAddressFilter,
+            Function<ProxiedAddresses, InetSocketAddress> clientAddressMapper,
```

Comment:
`Function<? super ProxiedAddresses, ? extends InetSocketAddress>`?

Reply:
Oops! Thanks for pointing out. 😭